doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2555,3 +2555,8 @@ the context to help inferring the types of the remaining arguments.
 
       Arguments ex_intro _ _ & _ _.
       Check (ex_intro _ true _ : exists n : nat, n > 0).
+
+Coq will attempt to produce a term which uses the arguments you
+provided, but in some cases involving Program mode the arguments after
+the bidirectionality starts may be replaced by convertible but
+syntactically different terms.
```

Comment:
I pushed this addition as I think we should mention the limitation in the doc. If someone familiar with Program could suggest an example that would be nice.